package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "webpack-wizard",
-  "version": "0.0.4",
+  "version": "0.0.5",
```

Comment:
Please do not update the version, this is a job for a person who publishes the npm package (myself).

Reply:
done